lib/testing/src/async/fake_async.dart

```diff
@@ -292,4 +301,10 @@ class _FakeTimer implements Timer {
   int get tick {
     throw new UnimplementedError("tick");
   }
+
+  /// Returns debugging information to try to identify the source of the
+  /// [Timer].
+  String get debugInfo =>
```

Comment:
Should we preserve the `StackTrace` to allow downstream code to decide how to filter stack frames?

Reply:
I think preserving it might be more trouble than it's worth.  Downstream code currently can filter by discarding the first line and using `StackTrace.fromString`; I probably should have explicitly made this part of the contract.